features/networking/operator.feature

```diff
@@ -121,44 +121,23 @@ Feature: Operator related networking scenarios
     """
 
   # @author bmeng@redhat.com
+  # @author zzhao@redhat.com
   # @case_id OCP-22202
   @admin
   @destructive
   Scenario: The clusteroperator should be able to reflect the realtime status of the network when a new node added
-    Given the master version >= "4.0"
+    Given I have an IPI deployment
```

Comment:
@zhaozhanqi Is it only specific to IPI?

Reply:
yes, this step will make sure this case running on ipi cluster since upi do not have machinesets I remember